chat/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+spring:
+  datasource:
+    driverClassName: com.mysql.jdbc.Driver
+    url: jdbc:mysql://allgame.fun:18300/chat-staging
+    username: chat-staging
+    password: egzqJnTvQ24855QzvpSGNqCfJnfAgH
```

Comment:
@wikiwu24 @MEtoCS  hello! please do not commit your database username and password. Could you please remove those?